README.md

```diff
@@ -1 +1 @@
-# react-todo-app
\ No newline at end of file
+# react-todo-app
```

Comment:
- README.md file should contain useful information and documentation. Kindly follow [the template](https://github.com/microverseinc/readme-template) on how the file should look like. 🖊️ 
- Also, don't forget to deploy the page using gh-pages using npm, GitHub Pages, Heroku or Netlify. 😉

Reply:
Thank you very much